orb-slot-ctrl/src/efivar/bootchain.rs

```diff
@@ -5,17 +5,30 @@
 //!
 //! Bits of interest are found in byte 4 for all efivars.
 
-use super::{is_valid_buffer, EfiVar, SLOT_A, SLOT_B};
+use super::{is_valid_buffer, EfiVar, EfiVarDb, EfiVarDbErr, SLOT_A, SLOT_B};
 use crate::Error;
 
-pub const PATH_CURRENT: &str =
-    "BootChainFwCurrent-781e084c-a330-417c-b678-38e696380cb9";
-pub const PATH_NEXT: &str = "BootChainFwNext-781e084c-a330-417c-b678-38e696380cb9";
+const PATH_CURRENT: &str = "BootChainFwCurrent-781e084c-a330-417c-b678-38e696380cb9";
```

Comment:
I know that you only changed their visibility, but it would be awesome to get a page reference to this in the EFI spec or wherever these values came from. Magic values with no documentation = sad face.

Reply:
agreed! i'll chase down where these come from and document them later :+1: